lib/plugins/docker/index.js

```diff
@@ -4,13 +4,25 @@ import preset from '../../pattern-preset';
 
 export default class Docker {
 
+  static clickHandler({ image }) {
+    let isOffical = true;
+    const imageName = image.split(':')[0];
+
+    if (image.includes('/')) {
+      isOffical = false;
+    }
+
+    return [
+      `https://hub.docker.com/${isOffical ? '_' : 'r'}/${imageName}`,
+    ];
+  }
+
   getPattern() {
     return preset('Docker');
   }
 
   parseBlob(blob) {
-    insertLink(blob.el, DOCKER_FROM, {
-      resolver: 'dockerImage',
+    insertLink(this, blob.el, DOCKER_FROM, {
```

Comment:
The idea of passing the plugin to `insertLink` was to avoid duplication of `pluginName: this.constructor.name` across all plugins. To avoid conflicts with other prop key names passed to `insertLink` the name is prefixed with an underscore. 

Maybe it's not the best idea, since it hides the actual behaviour and makes it just more complicated. Thanks for pointing me in the right direction. That's why i believe and love code reviews 👍